<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>571</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Преобразование изображений</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_scl_x">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_scl_y">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale_x">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X(pxl)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale_y">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y(pxl)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_trans">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Перенос</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_trans_y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dx">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>dX(pxl)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_trans_x">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dy">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>dY(pxl)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_undo">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>420</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale_kx">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KX</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale_ky">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>200</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KY</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_turn_y">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>300</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_turn_x">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_turn_y">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_turn_x">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>280</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>410</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_scale">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_turn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kx">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>200</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_ky">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>200</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale_kx_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>330</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>На            градусов</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_deg">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>330</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>480</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string> В начало </string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
